src/Entity/Funder.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\FunderRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=FunderRepository::class)
+ * @ORM\Table(name="funders")
+ */
+class Funder
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private int $id;
```

Comment:
All IDs are inconsistent type with their getter

Also, you can extract this property and the getter in a trait to reuse it everywhere if you wish (I usually create `src/Entity/Util/EntityIdTrait.php`)